site/logsdb-engine/css/demo.css

```
@@ -0,0 +1,559 @@
+  top: 0;
+}
+.log-time {
+  white-space: nowrap;
+  opacity: 0.7;
+}
+.log-svc code {
+  font-size: 0.7rem;
+}
+.log-body code {
+  font-size: 0.7rem;
+  word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)